packages/taco/integration-test/taco-client.test.ts

```diff
@@ -0,0 +1,255 @@
+import { beforeAll, describe, expect, test } from 'vitest';
+
+import { fromBytes, toBytes } from '@nucypher/shared';
+import {
+  EIP4361AuthProvider,
+  USER_ADDRESS_PARAM_DEFAULT,
+} from '@nucypher/taco-auth';
+import { ethers } from 'ethers';
+import { createPublicClient, http, LocalAccount } from 'viem';
+import { privateKeyToAccount } from 'viem/accounts';
+import { polygonAmoy } from 'viem/chains';
+import {
+  conditions,
+  initialize,
+  TacoClient,
+  TacoClientEthersConfig,
+  TacoClientViemConfig,
+  ThresholdMessageKit,
+} from '../src';
+import { DkgClient } from '../src/dkg';
+
+const RPC_PROVIDER_URL = 'https://rpc-amoy.polygon.technology';
+const ENCRYPTOR_PRIVATE_KEY =
+  '0x900edb9e8214b2353f82aa195e915128f419a92cfb8bbc0f4784f10ef4112b86';
+const CONSUMER_PRIVATE_KEY =
+  '0xf307e165339cb5deb2b8ec59c31a5c0a957b8e8453ce7fe8a19d9a4c8acf36d4';
+const DOMAIN = 'lynx';
+const RITUAL_ID = 27;
```

Comment:
Since Lynx is more a experimental network, the possibility that the network is malfunctioning when someone tries this test is greater than using tapir.
```suggestion
const DOMAIN = 'tapir';
const RITUAL_ID = 6;
```